pkg/qemu/qemu_driver.go

```diff
@@ -396,10 +397,8 @@ func (l *LimaQemuDriver) ListSnapshots(_ context.Context) (string, error) {
 	return List(qCfg, l.Instance.Status == store.StatusRunning)
 }
 
-func (l *LimaQemuDriver) GuestAgentConn(ctx context.Context) (net.Conn, error) {
-	var d net.Dialer
-	dialContext, err := d.DialContext(ctx, "unix", filepath.Join(l.Instance.Dir, filenames.GuestAgentSock))
-	return dialContext, err
+func (l *LimaQemuDriver) GuestAgentConn(context.Context) (net.Conn, error) {
+	return netutil.DialUnix("unix", filepath.Join(l.Instance.Dir, filenames.GuestAgentSock))
```

Comment:
Here again we dropped the context. This change is mostly dropping the context from many places instead of adding a helper. Lets keep the code the same.

Reply:
Sure. It is not exactly clear to me what cancellation actually does in the Unix socket case, but done.